components/Junior/structure.ts

```
@@ -9,27 +9,39 @@ export const getValidationSchema = (hasSubareas, requiresDate) => {
-        otherwise: (schema) => schema.nullable(),
-      }),
-    area: Yup.string()
-      .required("* Escolha uma opção por favor."),
+      .matches(
+        /^(https?:\/\/)(www\.)?linkedin\.com\/in\/[\w-]{3,}(\/)?(\?.*)?(#.*)?$/,
+        "Por favor, insira um link válido do LinkedIn (ex.: https://www.linkedin.com/in/seu-nome)."
+      )
+      .required("O campo LinkedIn é obrigatório."),
+    indicationLinkedin: Yup.string().when("indication", {
+      is: "sim",
+      then: (schema) =>
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)